substra/sdk/client.py

```diff
@@ -583,6 +592,7 @@ def add_compute_plan(self, data):
             }],
             "clean_models": bool,
             "tag": str
+            "metadata": list[str]
```

Comment:
I believe this is a leftover that you should remove.

Reply:
Indeed, I forgot to remove it!